pymc/tests/test_initvals.py

```diff
@@ -103,3 +109,33 @@ def test_basic(self):
         assert np.all(get_moment(rv).eval() == np.zeros((2, 4)))
         rv = pm.HalfFlat.dist(size=(2, 4))
         assert np.all(get_moment(rv).eval() == np.ones((2, 4)))
+
+    @pytest.mark.xfail(reason="Test values are still used for initvals.")
```

Comment:
Isn't that what this PR is about?

Reply:
No this PR is just to get some commits out of the upcoming initval framework PR, which is still blocked by `aeppl` integration.

These tests originally come from #4996 and are  regression tests for #4993 which needs the new initval framework to get fixed.